src/prisma/schema.prisma

```diff
@@ -2334,3 +2337,14 @@ model PersonaUserKyc {
   @@unique([userId, spaceId])
   @@index([userId, spaceId])
 }
+
+model ProposalEvaluationReview {
```

Comment:
please add indices to this table
i think reviewerId and evaluatiolnId can be unique, evaluationId needs an index for sure

Reply:
Updated. Thnx.